pom.xml

```diff
@@ -6,7 +6,7 @@
 
     <groupId>org.codice</groupId>
     <artifactId>usng4j</artifactId>
-    <version>0.2-SNAPSHOT</version>
+    <version>0.3-SNAPSHOT</version>
```

Comment:
❓ Was there a release of version 0.2 that warrants this version bump?

Reply:
Done, updated it back to 0.2.